src/Bridge/Doctrine/Orm/Filter/AbstractFilter.php

```diff
@@ -116,7 +116,7 @@ protected function isPropertyEnabled(string $property): bool
      */
     protected function isPropertyMapped(string $property, string $resourceClass, bool $allowAssociation = false): bool
     {
-        if ($this->isPropertyNested($property)) {
+        if ($this->isPropertyNested($property) && !$this->isPropertyNestedEmbedded($property, $resourceClass)) {
```

Comment:
Is this condition valid everytime? I mean, will it break current implementations if we put this inside `isPropertyNested` so that other filters also benefits from this? Maybe it's just the boolean filter that's impacted, but I doubt that.